lib/cookie.js

```diff
@@ -13,6 +13,9 @@ module.exports = class Cookie {
     this._expires = null
 
     if (originalMaxAge) {
+      if (cookie.expires) {
```

Comment:
Should we use `originalMaxAge` here?

Reply:
No. Please see my previous comment regarding expiry calculation bug. In this case, if `cookie.expiry` is set, it will be a string loaded from session store. So, it is converted to proper Date object using this if condition